src/ZoneServer/World/Actors/Characters/Components/CollectionComponent.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Melia.Zone.World.Items;
+using Melia.Shared.Data.Database;
+using Melia.Shared.Scripting;
+using Melia.Zone.Network;
+using Melia.Zone.Scripting;
+using Yggdrasil.Logging;
+using Yggdrasil.Scheduling;
+using Yggdrasil.Util;
+
+namespace Melia.Zone.World.Actors.Characters.Components
+{
+	/// <summary>
+	/// A character's collection manager.
+	/// </summary>
+	/// <remarks>
+	/// An important thing to note about the collection system is that Collections are account-level.
+	/// </remarks>
+	public class CollectionComponent : CharacterComponent
+	{
+		private readonly static TimeSpan AutoReceiveDelay = TimeSpan.FromMinutes(1);
```

Comment:
This isn't necessary for collections, right?

Reply:
You're right, that's a holdover from a component I used as a baseline, it's been taken out.